connectivity/check/context.go

```diff
@@ -1030,6 +1030,12 @@ func (ct *ConnectivityTest) CurlCommandParallelWithOutput(peer TestPeer, ipFam f
 }
 
 func (ct *ConnectivityTest) PingCommand(peer TestPeer, ipFam features.IPFamily) []string {
```

Comment:
nit: why not just `PingCommand(peer TestPeer, ipFam features.IPFamily, extraArgs ...string)`?